gateway-sp-comms/src/single_sp.rs

```diff
@@ -494,6 +494,11 @@ impl SingleSp {
             .and_then(expect_component_clear_status_ack)
     }
 
+    /// Request that the current system time.
+    pub async fn current_time(&self) -> Result<u64> {
+        self.rpc(MgsRequest::CurrentTime).await.and_then(expect_current_time)
```

Comment:
Nit: do we want to return a `Duration` here?

Reply:
We could if we're willing to bake the knowledge here that the SP is always returning ms?

Reply:
Yeah, I'm on the fence.  On one hand, it's not _required_, but a whole lot of SP firmware assumes it...

Reply:
Some options:

* Change this to assume millis
* Keep this as-is
* Change this to report both the raw value and interpreted as millis
* Rename this `current-time-raw` and add `current-time` which assumes millis

Reply:
I like option 4 (having both `current_time_raw` and `current_time`, assuming millis)

Reply:
Done in c29e480:

```
root@oxz_switch0:/tmp# FAUX_MGS=./john-faux-mgs pilot sp exec -e 'current-time-raw' BRM44220008
Nov 01 19:34:25.321 INFO creating SP handle on interface sidecar1, component: faux-mgs
Nov 01 19:34:25.322 INFO initial discovery complete, addr: [fe80::aa40:25ff:fe05:401%79]:11111, interface: sidecar1, component: faux-mgs
current time (raw): 857253425
root@oxz_switch0:/tmp# FAUX_MGS=./john-faux-mgs pilot sp exec -e 'current-time' BRM44220008
Nov 01 19:34:31.021 INFO creating SP handle on interface sidecar1, component: faux-mgs
Nov 01 19:34:31.022 INFO initial discovery complete, addr: [fe80::aa40:25ff:fe05:401%79]:11111, interface: sidecar1, component: faux-mgs
current time: 9days 22h 7m 39s 126ms
```